src/fparser/common/readfortran.py

```diff
@@ -1429,9 +1495,8 @@ def get_source_item(self):
                     label, line = extract_label(line)
                     name, line = extract_construct_name(line)
 
-                line, qc, had_comment = handle_inline_comment(line,
-                                                              self.linecount,
-                                                              qc)
+                line, qc, had_comment \
```

Comment:
Thanks, but I must admit I'm used to seeing the `=` on the first line with a line break after it.

Reply:
Oh, that change is probably a left-over from the previous version with checks around `handle_inline_comment`, where I had to add indentation to that line. I can shift the `=` to the first line.